src/js/view/View.js

```diff
@@ -0,0 +1,83 @@
+import { SELECTOR } from '../constants.js';
+import { $, $all } from '../utils/utils.js';
+// import template from '../templates.js';
+
+export default class View {
+  constructor() {
+    this.$app = $(SELECTOR.$APP);
+    this.$carNameInput = $(SELECTOR.$CAR_NAME_INPUT);
+    this.$carNameButton = $(SELECTOR.$CAR_NAME_BUTTON);
+    this.$racingCountInput = $(SELECTOR.$RACING_COUNT_INPUT);
+    this.$racingCountButton = $(SELECTOR.$RACING_COUNT_BUTTON);
+    this.$result = $(SELECTOR.$RESULT);
+    this.$racingResult = $(SELECTOR.$RACING_RESULT);
+  }
+
+  get result() {
+    return this.$result;
+  }
+
+  get racingResult() {
+    return this.$racingResult;
+  }
+
+  get carNameInput() {
+    return this.$carNameInput;
+  }
+
+  get racingCountInput() {
+    return this.$racingCountInput;
+  }
+
+  alertErrorMessage(message) {
+    alert(message);
+  }
+
+  initializeInput(clearElement, focusElement = clearElement) {
+    clearElement.value = '';
+    focusElement.focus();
+  }
+
+  racingCountInputVisibiled() {
+    $(SELECTOR.$INPUT_FORM_LAST_CHILD).style.display = 'block';
+  }
+
+  racingCountInputInvisibiled() {
+    $(SELECTOR.$INPUT_FORM_LAST_CHILD).style.display = 'none';
+  }
+
+  render(selector, template) {
+    selector.innerHTML = template;
+  }
+
+  bindEventListener(type, selector, callback) {
+    const children = [...$all(selector)];
+    const isTarget = (target) =>
+      children.includes(target) || target.closest(selector);
+
+    this.$app.addEventListener(type, (e) => {
+      if (!isTarget(e.target)) return;
+
+      e.preventDefault();
+      callback(e);
+    });
+  }
+
+  bindClickCarNameButton(callback) {
+    this.bindEventListener('click', SELECTOR.$CAR_NAME_BUTTON, () => {
+      callback();
+    });
```

Comment:
```suggestion
    this.bindEventListener('click', SELECTOR.$CAR_NAME_BUTTON, callback);
```
여기서 arrow function으로 감싸지 않아도 될 것 같지 않나요? 🙂

Reply:
넵 불필요한 arrow function인 것 같습니다.. ! 😅 
수정하도록 하겠습니다!